### PR TITLE
Fix bug that occurs when token metadata decimals is undefined.

### PR DESCRIPTION
**Pull request by albrow**
## Description

Title is self-explanatory. This is a bugfix in the token metadata parser.

## Checklist:

*   [ ] Prefix PR title with `[WIP]` if necessary.
*   [ ] Add tests to cover changes as needed.
*   [ ] Update documentation as needed.
*   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/pipeline/src/parsers/token_metadata/index.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ function parseMetamaskTrustedToken(resp: MetamaskTrustedTokenMeta, address: stri
     const trustedToken = new TokenMetadata();
 
     trustedToken.address = address;
-    trustedToken.decimals = new BigNumber(resp.decimals);
+    trustedToken.decimals = resp.decimals === undefined ? null : new BigNumber(resp.decimals);
```
Comment (albrow) [inline packages/pipeline/src/parsers/token_metadata/index.ts]: @xianny the only real difference is that we needed to check if `decimals` is `undefined` in the `parseMetamaskTrustedToken` function, not just in `parseZeroExTrustedToken` function.

Comment (xianny) [reply packages/pipeline/src/parsers/token_metadata/index.ts]: 👍 cool! sorry I missed this.

**Comment by albrow** [comment]
@xianny 

> would `resp.decimals` ever be null?

I haven't seen a case like that by we might as well make our solution more robust. Check out the latest changes.

**Comment by xianny** [comment]
Cool. Looks good!

**Pull request merged**